content/docs/setup/kubernetes/quick-start-alicloud-ack/index.md

```diff
@@ -0,0 +1,89 @@
+---
+title: Quick Start with AliCloud Kubernetes Container Service
+description: Quick Start instructions to setup the Istio service using AliCloud Kubernetes Container Service
+weight: 12
+keywords: [kubernetes,alicloud]
+---
+
+Quick Start instructions to install and run Istio in [AliCloud Kubernetes Container Service](https://cs.console.aliyun.com/) using Application Catalog.
+
+This Quick Start installs the current release version (0.8.0) of Istio and then deploys the [Bookinfo](/docs/guides/bookinfo/) sample
```

Comment:
Please remove the 0.8.0 part, otherwise the web page will be outdated quickly.
Please uee /docs/examples/bookinfo instead, the link was recently changed.